docs/dictionary/control_st/exit-repeat.lcdoc

```diff
@@ -34,9 +36,8 @@ condition is false. This example stops the <loop> when the user presses
 the mouse button:
 
     repeat with x = 1 to the number of cards
-    go card x
-    if the mouse is down then exit repeat -- bail out
-
+        go card x
```

Comment:
@livecodeali @livecodesam should we come up with an example that is moderately useful here? I can't for the life of me think why someone would want to do this...

Reply:
What about reading from a file in chunks?
```
constant kChunk = 1024
open file tFile for binary read
repeat
    read from file tFile for kChunk
    put the result into tResult
    if tResult is empty or tResult is "eof" then
         ProcessFileChunk it
    end if
    if tResult is not empty then
        exit repeat
    end if
end repeat
close file tFile
```